addons/im_livechat/static/tests/embed/livechat_service_tests.js

```diff
@@ -52,3 +54,39 @@ QUnit.test("previous operator prioritized", async () => {
     click(".o-livechat-LivechatButton");
     await contains(".o-mail-Message-author", { text: "John Doe" });
 });
+
+QUnit.test("Only necessary requests are made when creating a new chat", async (assert) => {
+    await startServer();
+    await loadDefaultConfig();
+    const linkPreviewDeferred = new Deferred();
+    await start({
+        mockRPC(route) {
+            if (!route.includes("assets")) {
+                assert.step(route);
+            }
+            if (route === "/mail/link_preview") {
+                linkPreviewDeferred.resolve();
+            }
+        },
+    });
+    await contains(".o-livechat-LivechatButton");
+    assert.verifySteps([
+        "/im_livechat/init",
+        "/web/webclient/load_menus", // called because menu_service is loaded in qunit bundle
+        "/mail/load_message_failures", // called because mail/core/web is loaded in qunit bundle
+    ]);
+    await click(".o-livechat-LivechatButton");
+    assert.verifySteps(["/im_livechat/get_session"]);
+    await insertText(".o-mail-Composer-input", "Hello!");
+    assert.verifySteps([]);
+    await triggerHotkey("Enter");
+    await contains(".o-mail-Message", { text: "Hello!" });
+    await linkPreviewDeferred;
+    assert.verifySteps([
+        "/im_livechat/get_session",
+        "/mail/init_messaging",
+        "/discuss/channel/fold",
```

Comment:
Introduced in https://github.com/odoo/odoo/pull/145941, will take care of it in a follow up pR